release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -578,6 +578,24 @@ If you are receiving pod security violations, see the following resources:
 
 * If necessary, you can set a custom admission profile on the namespace or pod by setting the `pod-security.kubernetes.io/enforce` label.
 
+[discrete]
+[id="ocp-4-13-oc-mirror-openshift-endpoint"]
```

Comment:
@skopacz1 just checking, could these technically be considered features/enhancements, and be listed in the appropriate section above (installation and upgrade)?

Reply:
I discussed this point with the developers and it seems like notable technical changes is the most appropriate location for these updates. 

If you happen to be a user who could not retrieve this data from GitHub for some sort of compliance/security reason, maybe this could be seen as an enhancement. But to most other users, it doesn't really change or improve any part of the process as far as I'm aware.

Also, at some point, should "upgrade" in that enhancement section be changed to update? I think "update" is the preferred term in the glossary for consistency.

Reply:
@bergerhoffer for 4.13 or all branches?

Reply:
@JoeAldinger just for 4.13 would be great